### ISSUE TITLE
Install a fire-safety-nerd approved keeper-opener for our Main Front Door (which has inconveniently been designated as a fire door) 

### ISSUE THREAD

**Issue by goatchurchprime** [DoESLiverpool/somebody-should#923]
The options are some kind of integrated magnetic door on the wall that releases at the same time as all the other magnetic locks when the fire alarm goes off... 

...or this battery powered device that has a microphone that hears the sound of the alarm and releases foot operated the catch.  Also contains dipswitches so it can be set to automatically shut at night.  
    https://www.safelincs.co.uk/dorgard-fire-door-retainers/

It's available over at screwfix, so we could fetch one today:
    https://www.screwfix.com/c/security-ironmongery/fire-door-retainers/cat840324

This is an embarrassingly lovely IoT device used out in the real world that we really ought to have known about. How do they get a microphone activated device to last a whole year on one set of batteries?  What kind of powerful solenoid is able release a spring-loaded foot operated catch?  The answers will be inside this consumer device.

**Comment by goatchurchprime** [comment]
Should I go out to Screwfix and get one of these, so we can hold the door open legally?  It's about £120.

I'm thinking of mounting the old doorbot welcome screen beside the light switch just on the right hand wall on the inside of the front door, where it will be seen by anyone outside (and draw them in) -- if the door is propped open.

**Comment by johnmckerrell** [comment]
I was generally sold on these but something else that just occurred to me was that it’s going to affect heating the space if we have the door open all the time.

**Comment by skos-ninja** [comment]
It might make more sense to define a policy for when the door should be left open rather than defaulting to all the time.

I know that during maker night it makes more sense to have the door open and welcoming but during just a normal business day it might not.

**Comment by DoESsean** [comment]
I think attempts were made at this which failed, and it's not been revisited, so I'm closing it for now and adding the Holding out for a Hero tag.

**Comment by amcewen** [comment]
Given particularly that it's the season for propping the door open, we should fit this.  I think it stalled because sometimes it released when it shouldn't, but I don't think it's the end of the world if we have to open the door every now and then because it's released accidentally.

**Comment by amcewen** [comment]
Had an initial look at fitting it this evening.

If we fit it to the inside of the door (the most obvious side to use) then it would foul on the carpet on this side (because that's got a sheet of plywood under it).  However, there's therefore quite a big gap under the door when it's open, so we could fit the floorplate on a piece of 9mm ply to raise it off the floor, which would then stop it fouling when closed.

Ran out of time for that tonight.

**Comment by amcewen** [comment]
I've cut some shims and installed the floorplate.  I stuck the [design file](https://github.com/DoESLiverpool/somewhere-safe/blob/master/Workshop/LaserCut/DoorStopFootplate.svg) somewhere-safe in case we need it again when we fit one to the events room door too.

**Comment by amcewen** [comment]
I've fitted the doorstop to the main door now.  We can see how well it works, and if there aren't any problems then we should buy one and install it on the Events Space door too.

**Comment by johnmckerrell** [comment]
Just making a note that if we’re ordering more I’d imagine we’d want a second for the main room door(?) and one for room 29 too. 

--

**Comment by johnmckerrell** [comment]
The opener has been beeping for the past few days, I assume it's a battery issue. I've removed the current batteries and we'll need to buy some more C type batteries

@amcewen had the batteries been left in since it was bought or have they managed to run down in just a few days?

**Comment by amcewen** [comment]
Was there something wrong with the C cell batteries in the Batteries drawer?  :-D  I had three C cells at home that I didn't need any more - two are in the opener, and I've put the third with the others in the batteries drawer by the VR computer.

The batteries had been left in it since it was bought, so seem to last about 10 months

**Comment by johnmckerrell** [comment]
They’re probably the ones I took out!

--

**Comment by jackie1050** [comment]
There were a few left over from the drum lights - could be those?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by amcewen** [comment]
Screwfix don't have one in stock over the road, so I've ordered [one from their website](https://www.screwfix.com/p/union-doorsense-j-8755a-acoustic-release-hold-open-unit-black/2464t) instead.  Should arrive tomorrow, then we can fit it to the Events Room door.

**Comment by amcewen** [comment]
The second one arrived and I've fitted it this morning, so I'm going to close this.  I've added a note to #922 to remind us we might want to fit one to the Room 29 when that gets a closer.

**Comment by amcewen** [comment]
There was a fire alarm test today, so I propped both doors open and the closers worked perfectly.

**Comment by johnmckerrell** [comment]
I noticed during the repair café that it did close a number of times, I wonder if that's what we experienced previously about it releasing incorrectly. Seemed like background noise was enough to activate it. Was there an option to change the sensitivity?
